deploy/kubernetes/base/controller/cluster_setup.yaml

```diff
@@ -143,6 +146,9 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
+  - apiGroups: ["storage.k8s.io/v1alpha1"]
```

Comment:
I think the apiGroups  should just have "storage.k8s.io" and "v1alpha1" isn't needed. Same comment below.

Reply:
Done